yara-sys/build.rs

```diff
@@ -5,6 +5,13 @@ fn main() {
     bindings::add_bindings();
 }
 
+#[macro_export]
+macro_rules! cargo_rerun_if_env_changed {
+    ($env_name:tt) => {{
+        println!("cargo:rerun-if-env-changed={}", $env_name);
+    }};
+}
```

Comment:
Why a macro?
Don't this would do the same thing?
```rust
fn cargo_rerun_if_env_changed(env_name: &str) {
    println!("cargo:rerun-if-env-changed={}", env_name)
}
```